assets/src/application/ui/layout/entityActionsMenu/useEntityActionsService.ts

```diff
@@ -0,0 +1,28 @@
+import { filter } from 'ramda';
+
+import { useSubscriptionService } from '@appServices/subscription';
+import { EntityActionsService, EntityActionsServiceHook } from './types';
+import { serviceName as service } from './constants';
+
+type EAS = EntityActionsService;
+type EASH = EntityActionsServiceHook;
+
+const useEntityActionsService: EASH = (domain) => {
+	const { getSubscriptions: getServiceSubscriptions, ...restServices } = useSubscriptionService({
+		domain,
+		service,
+	});
+
+	const getSubscriptions: EAS['getSubscriptions'] = (options = {}) => {
+		const { entityType } = options;
+		const allSubscriptions = getServiceSubscriptions();
+		if (entityType) {
+			return filter(({ options }) => entityType === options.entityType, allSubscriptions);
+		}
+		return allSubscriptions;
+	};
```

Comment:
This overrides the default behavior of application service to make use of subscription filtering by entity type.